slides.typ

```diff
@@ -94,74 +178,33 @@
 #let slides(
     title: none,
     author: none,
+    subtitle: none,
     short-title: none,
     short-author: none,
     date: none,
-    color: teal,
-    document_body
+    theme: slides-default-theme(),
+    body
 ) = {
-    show heading.where(level: 1): h => {
-        section.update(h.body)
-    }
-
-    show heading.where(level: 2): h => {
-        pagebreak(weak: true)
-        logical-slide.step()
-        h
-    }
-
     set text(
         size: 25pt,
     )
 
-    let decoration(position, body) = {
-        let border = 1mm + color
-        let strokes = (
-            header: ( bottom: border ),
-            footer: ( top: border )
-        )
-        block(
-            stroke: strokes.at(position),
-            width: 100%,
-            inset: .3em,
-            text(.5em, body)
-        )
-    }
-
     set page(
         paper: "presentation-16-9",
```

Comment:
Configurable as well.

Reply:
Agreed, same question as with the font size though. Also would we just ask for any paper size? Or only accept the strings `"16-9"` and `°4-3"`?

Reply:
Probably accept any size just in case